src/test/java/co/rsk/federate/FederationProviderFromFederatorSupportTest.java

```diff
@@ -121,7 +121,7 @@ void getActiveFederation_erp_federation() {
         ActivationConfig.ForBlock configMock = mock(ActivationConfig.ForBlock.class);
         when(configMock.isActive(RSKIP123)).thenReturn(true);
 
-        Federation expectedFederation = createErpFederation(
+        Federation expectedFederation = createNonStandardErpFederation(
             getFederationMembersFromPks(1, 1000, 2000, 3000, 4000, 5000),
```

Comment:
No big deal, but I think these values for a btc private key look unrealistic. I would suggest having valid btc private keys, probably declared as static constant at the beginning of this class. To keep everything as real as possible.